keras_cv/models/backbones/densenet/densenet_backbone_presets.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2023 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""DenseNet model preset configurations."""
+
+backbone_presets_no_weights = {
+    "densenet121": {
+        "metadata": {
+            "description": (
+                "DenseNet model with 121 layers where the batch normalization "
```

Comment:
Do we need any qualifications after "layers"? I think this is leftover from the ResNet copy.

Reply:
Decided to let it stay there for more information. I agree it does not provide more comparative context between the models. Removed.